public/app/core/components/LayoutSelector/LayoutSelector.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function LayoutSelector({ mode, onLayoutModeChanged }) {
```

Comment:
some typings for the params of this function would be great. mode could be an enum